worldedit-core/src/main/java/com/sk89q/worldedit/regions/EllipsoidRegion.java

```diff
@@ -409,7 +409,7 @@ public void filter(IChunk chunk, Filter filter, ChunkFilterBlock block, IChunkGe
             int yTopFull = Math.min(255, cy + diffYFull);
 
             if (yBotFull == yTopFull || yBotFull > yTopFull) {
-                System.out.println("aa");
+                System.out.println("aa"); //why do we print "aa" here?
```

Comment:
Looks like some left over debug?

Reply:
Yeah, that's what I thought but what could it mean and is it safe to delete?

Reply:
I'd say so.

Reply:
```suggestion
```